Cargo.toml

```diff
@@ -20,6 +20,7 @@ members = [
     # Authenticator CLI,
     "authenticator-cli",
     # Tutorial / Example sites.
+    "tutorial/server/cli",
```

Comment:
Given you intend to use this as a server process, it probably shouldn't be in the tutorial :)

Reply:
Perhaps a [bin] in webauthn-rs itself?

Reply:
authenticator-cli is for interacting with CTAP2 devices specifically, so I think that's not right here.

I think that webauthn-server-proxy-bin or something else? What's a name that signals that a server calls out to this for processing those operations?

Reply:
So move `tutorial/server/cli/` to `webauthn-server-proxy-bin/`?

Should I also change `name = "webauthn-cli"` to `name = "webauthn-server-proxy"` in what's now `tutorial/server/Cargo.toml`?

Sorry if I'm slow to understand.

Reply:
Lets decide on the name first before we move it - but once we choose a good name, then "yes" to both of those steps :)

Reply:
I like `webauthn-server-proxy`.  I'll wait for your decision.

Reply:
Had a chat externally to micolous, we think `webauthn-rp-proxy` is the right name here if that's okay?

Reply:
That's a good name.  Done.

Reply:
Thanks